src/mcp/tools/firestore/list_collections.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { listCollectionIds } from "../../../gcp/firestore.js";
+import { NO_PROJECT_ERROR } from "../../errors.js";
+
+export const list_collections = tool(
+  {
+    name: "list_collections",
+    description:
+      "Retrieves a list of collections from a Firestore database in the current project.",
```

Comment:
Should this be:

`Retrieves a list of collections from the root of a Firestore database in the current project`

Would we consider listing document level collections?

Reply:
Added an optional `document_path` argument.